templates/main.template

```diff
@@ -342,14 +416,12 @@ Resources:
     Type: AWS::CloudFormation::Stack
     DependsOn: ProductionVpcTemplate
     Properties:
-      TemplateURL:
-        !Sub
-        - https://${QSS3BucketName}.${QSS3Region}.amazonaws.com/${QSS3KeyPrefix}submodules/quickstart-compliance-common/templates/vpc-management.template
-        - QSS3Region:
-            !If
-            - GovCloudCondition
-            - s3-us-gov-west-1
-            - s3
+      TemplateURL: https://cloudformationlctemplates.s3.amazonaws.com/vpc-management.template_LCQS
```

Comment:
See above

Reply:
Really just clarifying security group descriptions and restricting the security group from mgmt VPC to Prod VPC from only the bastion IP